src/views/WatchListView.vue

```diff
@@ -40,54 +40,47 @@
           <mdicon name="dice-multiple-outline" />
```

Comment:
I'll be honest, I'm not sure how the alignment worked before. To fix, probably best to remove the 'float-left' from the buttons and surround them in their own div. Then the buttons and poster card grid should be on different lines. From there you can add whatever additional styling you want (justify etc.)